Makefile

```diff
@@ -51,6 +51,15 @@ check_flake8:
 check_isort:
 	tox -e isort -- --check $(CHECK_SYNTAX_FILES)
 
+## Starts a postgres container in the background
+postgres:
+	docker run -itd --rm --name dab_postgres -p 55432:5432 -e POSTGRES_USER=dab -e POSTGRES_PASSWORD=dabing -e POSTGRES_DB=dab_db postgres:13
```

Comment:
I don't think `--interactive` and `--tty` flags are actually needed when running postgresql server.

Reply:
Confirming - this suggestion is to replace `-itd` with `-d`